src/server/routes/__tests__/submissions.unit.spec.js

```diff
@@ -223,7 +223,7 @@ describe('submissions.unit', () => {
   })
 
   describe('GET /submissions/{submissionId}', () => {
-    const getSubmissionHandler = routes[2].options.handler
+    const getSubmissionHandler = routes[3].options.handler
```

Comment:
rather than scattering these through the code, I think it'd be better to extract them all at the top of the file (that's the pattern I normally use), something like this:
```
import routes from '../submissions.js'
const [
  { options: { handler: postSubmissionHandler } },
  { options: { handler: getByContactIdAndSeasonSubmissionHandler } },
  { options: { handler: getByActivitiesSubmissionHandler } }
] = routes
```